module/platform/darwin/HardwareIO/README.md

```diff
@@ -1,40 +1,36 @@
-Darwin Hardware I/O
+NUgus Hardware I/O
 ===================
 
 ## Description
 
-This module is responsible for communicating with the Darwin robot's CM740
-controller.
+This module is responsible for communicating with the NUgus' CM740 controller.
 
 ## Usage
 
-Darwin Hardware I/O connects at startup to the Darwin controller located on
+NUgus Hardware I/O connects at startup to the NUgus controller located on
 `/dev/usbTTY0`. If this does not succeed an exception is thrown and startup is
```

Comment:
[udev rules](https://github.com/NUbots/NUbots/blob/b411e14fa76bf36d742531cb83d42bb7a42490d3/doc/ArchInstall/arch-chroot_install.sh#L205-L216) are set up on the robot to automagically map the CM740 device to `/dev/CM740`
```suggestion
`/dev/CM740`. If this does not succeed an exception is thrown and startup is
```